runtime/autoload/health/provider.vim

```diff
@@ -258,71 +258,56 @@ function! s:check_python(version) abort
   call health#report_start('Python ' . a:version . ' provider (optional)')
 
   let pyname = 'python'.(a:version == 2 ? '' : '3')
-  let pyenv = resolve(exepath('pyenv'))
-  let pyenv_root = exists('$PYENV_ROOT') ? resolve($PYENV_ROOT) : ''
+  let python_exe = ''
   let venv = exists('$VIRTUAL_ENV') ? resolve($VIRTUAL_ENV) : ''
   let host_prog_var = pyname.'_host_prog'
   let loaded_var = 'g:loaded_'.pyname.'_provider'
-  let python_bin = ''
```

Comment:
Leaving this name as it was would avoid a lot of churn.

Reply:
I intend to move every bigger `if`/`else` branch in `s:check_python()` to its own branch anyway in upcoming patches.

It's probably better to ignore the first two patches during review right now. :>

Reply:
FWIW: I would also have liked to not have this renaming while investigating changes around that code.